ipam/client/backends/phpipam.py

```diff
@@ -218,6 +218,54 @@ def add_top_level_subnet(self, subnet, description):
 
         return True
 
+    def add_subnet(self, subnet, parent_subnet, description):
+        """
+        Add a subnet if can be inserted in parent subnet.
+        """
+        subnet_len = subnet.prefixlen
+        children_subnets = list(parent_subnet.subnets(new_prefix=subnet_len))
```

Comment:
This can be very long, especially if using IPv6.

Reply:
Please check the latest commits :+1: